### ISSUE TITLE
`ocm-share-list-received` results in 500 error

### ISSUE THREAD

**Issue by michielbdejong** [pondersource/sciencemesh-nextcloud#212]
This is happening right now for einstein on mesh.pondersource.com.
Investigating.

**Comment by michielbdejong** [comment]
```
{"reqId":"NT9jbZ3B2Tu3x8R12e93","level":3,"time":"2023-06-13T10:47:45+00:00","remoteAddr":"138.68.152.232","user":"--","app":"index",
"method":"POST",
"url":"\/index.php\/apps\/sciencemesh\/~einstein\/api\/ocm\/ListReceivedShares",
"message":"Caused by: {\"Exception\":\"PDOException\",\"Message\":
\"SQLSTATE[42S22]: Column not found: 1054 Unknown column 'share_type' in 'where clause'\",
\"Code\":\"42S22\",\"Trace\":\"#0 \\\/var\\\/www\\\/html\\\/lib\\\/composer\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Driver\\\/PDOStatement.php(117): PDOStatement->execute()\\n#1 \\\/var\\\/www\\\/html\\\/lib\\\/composer\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Connection.php(1304): Doctrine\\\\DBAL\\\\Driver\\\\PDOStatement->execute()\\n#2 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/DB\\\/Connection.php(191): Doctrine\\\\DBAL\\\\Connection->executeQuery()\\n#3 \\\/var\\\/www\\\/html\\\/lib\\\/composer\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Query\\\/QueryBuilder.php(212): OC\\\\DB\\\\Connection->executeQuery()\\n#4 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/DB\\\/QueryBuilder\\\/QueryBuilder.php(141): Doctrine\\\\DBAL\\\\Query\\\\QueryBuilder->execute()\\n#5 \\\/var\\\/www\\\/html\\\/apps\\\/sciencemesh\\\/lib\\\/ShareProvider\\\/ScienceMeshShareProvider.php(472): OC\\\\DB\\\\QueryBuilder\\\\QueryBuilder->execute()\\n#6 \\\/var\\\/www\\\/html\\\/apps\\\/sciencemesh\\\/lib\\\/Controller\\\/RevaController.php(1076): OCA\\\\ScienceMesh\\\\ShareProvider\\\\ScienceMeshShareProvider->getReceivedShares()\\n#7 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(169): OCA\\\\ScienceMesh\\\\Controller\\\\RevaController->ListReceivedShares()\\n#8 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(89): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController()\\n#9 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/AppFramework\\\/App.php(99): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch()\\n#10 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/AppFramework\\\/Routing\\\/RouteActionHandler.php(47): OC\\\\AppFramework\\\\App::main()\\n#11 \\\/var\\\/www\\\/html\\\/lib\\\/private\\\/Route\\\/Router.php(344): OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler->__invoke()\\n#12 \\\/var\\\/www\\\/html\\\/lib\\\/base.php(913): OC\\\\Route\\\\Router->match()\\n#13 \\\/var\\\/www\\\/html\\\/index.php(54): OC::handleRequest()\\n#14 {main}\",\"File\":\"\\\/var\\\/www\\\/html\\\/lib\\\/composer\\\/doctrine\\\/dbal\\\/lib\\\/Doctrine\\\/DBAL\\\/Driver\\\/PDOStatement.php\",\"Line\":117}"}
```

**Comment by michielbdejong** [comment]
https://github.com/pondersource/nc-sciencemesh/blob/4a31083/lib/ShareProvider/ScienceMeshShareProvider.php#L467

**Comment by michielbdejong** [comment]
Fixed in [4e617c3](https://github.com/pondersource/nc-sciencemesh/commit/4e617c3f4c03eac03367ced9f302995262dc885d)
